cmd/release.go

```diff
@@ -39,6 +47,8 @@ var releaseCmd = &cobra.Command{
 // init prepares cobra flags
 func init() {
 	Promu.AddCommand(releaseCmd)
+
+	releaseCmd.Flags().IntVar(&allowedRetry, "retry", 2, "Number of retry to perform when upload fails")
```

Comment:
"Numer of retry" -> "Number of retries"

(and end flag descriptions with a period)